src/controllers/passwordReset.js

```diff
@@ -0,0 +1,80 @@
+import bcrypt from 'bcryptjs';
+
+import UserService from '../services/passwordResetService';
+
+import Response from '../utils/Response';
+
+import PasswordEmail from '../utils/Mailer';
+
+import Template from '../utils/Template';
+
+
+const { successResponse, errorResponse } = Response;
+
+const { getUser, updateUserPassword } = UserService;
+
+const { sendEmail } = PasswordEmail;
+/**
+ *
+ *
+ * @class UserController
+ */
+class UserController {
+/**
+ *
+ *
+ * @static
+ * @param {object} req
+ * @param {object} res
+ * @returns{object} User
+ * @memberof UserController
+ */
+  static async getAUser(req, res) {
+    const { email } = req.body;
+    try {
+      const AUser = await getUser(email);
+      if (!AUser) {
+        return res.status(404).json(errorResponse(`Cannot Find User With Email: ${email}`));
+      }
+      const { id } = AUser;
+      try {
```

Comment:
I see that we have a `try-catch` statement inside another `try-catch` statement. We should stick to one `try-catch` statement. Any error thrown will be caught by the since catch block.